borsh/tests/schema/test_ip_addr.rs

```diff
@@ -0,0 +1,35 @@
+use crate::common_macro::schema_imports::*;
+
+#[cfg(feature = "std")]
+use std::net::IpAddr;
+
+#[test]
+fn ip_addr_schema() {
+    let actual_name = IpAddr::declaration();
+    let mut actual_defs = schema_map!();
+    IpAddr::add_definitions_recursively(&mut actual_defs);
+
+    assert_eq!("IpAddr", actual_name);
+    assert_eq!(
+        schema_map! {
```

Comment:
please, replace `schema_map!` with `insta::assert_snapshot!(format!("{:#?}", defs));` similar to how it's done here
https://github.com/near/near-sdk-rs/pull/1212/commits/bd45f91862d51816c3b1459431383d98e8086304